### DISCUSSION TITLE
Tool not working

### DISCUSSION THREAD

**Discussion by Wasifsheikh01** [Alygnt/Clifty#17] [Q&A] [answered]
Can anyone provid video tutorial please???

**Comment by Alygnt** [comment, marked as answer]
Tool is perfectly fine and for tutorial just scroll down and see the screenshot page if you need full video tutorial i will post it

**Reply by Wasifsheikh01** [reply]
it will be great if you can post tutorial video!!!

**Reply by Alygnt** [reply]
Sure but it might take time

**Reply by lovelysexly7** [reply]
does the url work outside, in normal browser

**Reply by Alygnt** [reply]
Yes it does

**Reply by lovelysexly7** [reply]
is it localhost or ngrok because ngrok not working for me

**Reply by Alygnt** [reply]
If you want to use the link to devices connected to your wifi or your mobile hotspot, Use localhost l or If you want to use it worldwide use ngrok. Download ngrok > Signup it's website and paste the authtoken to make it fully potential and If any any other queries let me know

**Reply by miko-saputra** [reply]
Hai cara melihat hasil gambar gimana punya saya kok gak bisa
